client/ayon_core/pipeline/farm/pyblish_functions.py

```diff
@@ -1,20 +1,17 @@
-import os
 import copy
+import os
 import re
 import warnings
 from copy import deepcopy
 
 import attr
 import ayon_api
 import clique
-
-from ayon_core.pipeline import (
-    get_current_project_name,
-    get_representation_path,
-)
 from ayon_core.lib import Logger
-from ayon_core.pipeline.publish import KnownPublishError
+from ayon_core.pipeline import get_current_project_name, get_representation_path
+from ayon_core.pipeline.create import get_product_name
 from ayon_core.pipeline.farm.patterning import match_aov_pattern
```

Comment:
Could this be relative import?

Reply:
PEP-8 state that:
> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path)

Relative are perfectly valid in cases, where it can (in complex scenarios) improve readability. While these imports are definitely heading that way, I don't think that they are there yet :)